polaris-react/src/components/Navigation/types.ts

```diff
@@ -0,0 +1,67 @@
+import type {ReactNode} from 'react';
+
+import type {TooltipProps} from '../Tooltip';
+import type {IconProps} from '../Icon';
+
+export interface ItemURLDetails {
+  url?: string;
+  matches?: boolean;
+  exactMatch?: boolean;
+  matchPaths?: string[];
+  excludePaths?: string[];
+  external?: boolean;
+}
+
+export interface ItemProps extends ItemURLDetails {
+  icon?: IconProps['source'];
+  badge?: ReactNode;
+  label: string;
+  disabled?: boolean;
+  accessibilityLabel?: string;
+  selected?: boolean;
+  exactMatch?: boolean;
+  new?: boolean;
+  addLine?: boolean;
+  addHoverLine?: boolean;
+  addHoverPointer?: boolean;
+  onMouseEnter?(label: string): void;
+  onMouseLeave?(): void;
```

Comment:
These props are the only change made for this interface

Reply:
I find the `addLine` prop names a bit confusing 

What about something like `showVerticalLine` `showVerticalPointer` etc...?

Reply:
Love it. I'll change it